Gruntfile.js

```diff
@@ -654,7 +654,8 @@ module.exports = function( grunt ) {
 		"test:functional",
 		"uglify",
 		"compare_size",
-		"commitplease"
+		"commitplease",
+        "update-authors"
```

Comment:
@jzaefferer i don't know why but the indentation appeared correctly on my text editor (Adobe Brackets), however i thought it was a editor bug and added a tab.

Indentation looks correct at [github diff here](https://github.com/jquery/globalize/pull/555/files) but [continuous-integration/travis-ci/pr](https://travis-ci.org/jquery/globalize/builds/92048148) fails because of bad indentation.